### DISCUSSION TITLE
Mouse Over

### DISCUSSION THREAD

**Discussion by natequreshi** [Hofknecht/SystemTrayMenu#522] [Ideas]
Ther was a Tool SE Tray Menu, STM is way better, but only lacking feature of it show menu on hover or option to enable it, it should have a docking type of facility too, I missing RocketDock.

**Comment by topeterk** [comment]
I think you have to be a little more specific what you actually mean. Menus should open on hover (you man moving the mouse over an item that a folder will be expanded) or do you talk about the trayicon?

And what kind of docking type (what does facility mean in this context?)? The menu opens on the configured location which will automatically reposition when the screen resolution or taskbar configuration changes. But it feels like you are not talking about this type of "docking", right?
I don't know RocketDock, but from a quick web search, when this is still about this docking type, I believe you want a floating menu? You can configure a custom position for the main menu in the settings and you can keep it open using the pin icon on the buttom right. Not exactly the same but at least something you can try.

**Reply by natequreshi** [reply]
hi, i refered a program Se Tray Menu which show menu when we mouse over the icon in tray, that program is no more usable, where I am using SyMenu which also no offering this feature, I found this STM more usable but just this feature needs to be added I don't know if you guys want to or not its just my thought that if program show program icons when mouse over progrma icon in tray.

I just mentioned RocketDock, no need to discuss that, but as STM is build and going to be a good utility any thing can be added like docking into start menu or on desktop.

**Reply by topeterk** [reply]
Ah I see now, that's quite interessting. Yes, I think we might have a look at that.
